tests/ecosystem/upgrade/test_noobaa.py

```diff
@@ -57,7 +58,10 @@ def test_fill_bucket(
         )
 
     mcg_obj_session.check_if_mirroring_is_done(bucket.name)
-    assert bucket.status == constants.STATUS_BOUND
```

Comment:
Regarding the three assertion messages (lines 60, 70, 80) - I feel they're rather redundant, considering `bucket.status` already logs the retrieved status, and the value of `constants.STATUS_BOUND` will be logged as part of the assertion error (while it is also accessible by checking the constants file, of course)
![image](https://user-images.githubusercontent.com/19227853/84402604-504d0a80-ac0d-11ea-8dfc-470e4e12d22b.png)

Reply:
Good point, changed